src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -45,7 +45,8 @@ static ModuleUtils()
                 | (FileAttributes)FILE_ATTRIBUTE_RECALL_ON_DATA_ACCESS
                 | (FileAttributes)FILE_ATTRIBUTE_RECALL_ON_OPEN;
 
-            return;
+			ApplicationInsightsTelemetry.SendUseTelemetry("PSModuleAutoLoadSkipOfflineFiles", "SkipOffLine");
```

Comment:
The `ModuleUtils` class is used in core component of the engine -- command discovery and module loading, so this static constructor will always run in a PowerShell session, which makes it not really useful to collect this telemetry in my opinion. What do you think?